src/shutdown/umount.c

```diff
@@ -151,18 +151,20 @@ int mount_points_list_get(const char *mountinfo, MountPoint **head) {
                         remount_flags = (remount_flags|MS_REMOUNT|MS_RDONLY) & ~MS_BIND;
                 }
 
-                m = new0(MountPoint, 1);
+                m = new(MountPoint, 1);
                 if (!m)
                         return log_oom();
 
+                *m = (MountPoint) {
+                        .remount_options = TAKE_PTR(remount_options),
```

Comment:
Not like that'll ever matter, but this gets leaked if the strdup futher down fails, no?

Reply:
nope, the `m` variable is marked with `_cleanup_`

Reply:
Yes, `m` is freed, but `m->remount_options` would leak.

Reply:
oh, right. it's a simple `_cleanup_free_` indeed